Reflections/ReflectionWeek4.md

```diff
@@ -0,0 +1,17 @@
+1. For this week I didn't accomplish that much as I have been busy with midterms.
+- I was able to get a working screen color changing slider so the user can actually make the design of their app be whatever they want because I wanted the color wheel to represent the different colors the app can be.
+- I didn't get finished but I recently started using the free to use editing application because I didn't have the money for Adobe After Effects. I used this to begin creating the animation for the logo when the app starts up.
+- I didn't get to implement the search filter yet because I didn't fully understand how Yelp did it but it was I who was getting mixed up.
+
+2. The biggest challenge that I faced this week was trying to stay on task while having 2 other midterms to study for, so I didn't get as much done as I wanted to. I am going to try and put it into overdrive this weekend and work extra hard.
+
+3. Now that I have plan to use most of my weekend to work on this app I think I will have pretty much the same goals as listed above.
+- I plan to work on creating an animation of the logo through the free to use application, Davinci Resolve. This should only take a couple of hours, depending on how long it will take my computer to process it and output the final result.
+- Then I want to work on creating the search filter inside of the filters portion of my app. This also should only take a couple of hours to implement.
+- Finally I want to begin working on D5 to get some user testing done for my application to see if I can find any bugs with my app. From the looks of it this documentation is pretty lengthy so it might take me a couple of days to a week to complete it all the way.

```

Comment:
The Testing document is long but you can cross out some sections you do not need like the game testing part (since you are not doing a game) let me know if you need help understanding any of the testing methods.